examples/acs-kubernetes/README.md

```diff
@@ -0,0 +1,104 @@
+# Deployment of Kubernetes cluster in the Azure Container Service
+
+Create a Kubernetes cluster in Azure using the Azure Container Service. This is based on the [101-acs-kubernetes](https://github.com/Azure/azure-quickstart-templates/tree/master/101-acs-kubernetes) Azure Quick Start Template.
+
+## Pre-requisites
+
+
+### Setting up Terraform Access to Azure
+
+To enable Terraform to provision resources into Azure, you need to create two entities in Azure Active Directory (AAD) - AAD Application and AAD Service Principal. [Azure CLI 2.0](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) allows you to quickly provision both by following the instructions below. 
```

Comment:
could I suggest this is a link to https://www.terraform.io/docs/providers/azurerm/index.html#creating-credentials - in-case this changes in the future so we have one less place to update?